docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/anemia_impairment/index.rst

```diff
@@ -4,6 +4,172 @@
+  :widths: 15, 15, 15, 15
+  :header-rows: 1
+
+  * - Anemia Severity
+    - Disability Weight
+  * - Mild
+    - 0.004
+  * - Moderate
+    - 0.052
+  * - Severe
+    - 0.149
+
+.. warning::
+
+  For the early and late neonatal age groups, the post-neonatal anemia prevalence was simply copied as the anemia prevalence for these age-groups rather than direct estimation of anemia among these age groups in GBD 2019.
+
+Causal Attribution
+++++++++++++++++++
+
+While the anemia envelope represents the total prevalence of anemia, the causal attribution process allows for estimation of what conditions *cause* that anemia. With some exceptions (see below_), two inputs were required for each cause included in the causal attribution process, 1) the cause prevalence (generated from other GBD processes), and 2) the cause-specific hemoglobin shift. Cause-specific hemoglobin shifts were derived from published studies that compared hemoglobin concentrations among diseased and non-diseased populations; these shifts were meta-analyzed for use in GBD 2019.
+
+.. _below:
+
+Notably, there were several causes that were not assigned specific hemoglobin shifts, including dietary iron deficiency; other infectious diseases; other neglected tropical diseases; other endocrine, nutrition, blood, and immune disorders; and other hemoglobinopathies and hemolytic anemias. Instead, the residual anemia envelope (with an enforced minimum 10%) were assigned to these causes in a manner analogous to fixed proportion redistribution.
+
+A complete list of the causes included in the causal attribution process for anemia include: 
+
+  P. falciparum parasitaemia without clinical malaria; P. vivax parasitaemia without clinical malaria; Clinical malaria; Schistosomiasis; Hookworm disease; Other neglected tropical diseases; Maternal haemorrhage; Vitamin A deficiency (under 15 years only); Other infectious diseases; Peptic ulcer disease; Gastritis; Stage III chronic kidney disease; Stage IV chronic kidney disease ; Stage V chronic kidney disease; End stage renal disease; Uterine fibroids; Menstrual disorders; Other haemoglobinopathies and haemolytic anaemias; Other endocrine, nutrition, blood, and immune disorders; G6PD deficiency; Hemizygous G6PD deficiency; Beta-thalassaemia major; Beta-thalassaemia trait; Haemoglobin E trait; Haemoglobin E/beta-thalassaemia; Haemoglobin H disease; Homozygous sickle cell and severe sickle cell/beta-thalassaemia parent; Haemoglobin SC disease; Mild sickle cell/beta-thalassaemia; Sickle cell trait; HIV; Cirrhosis and other chronic liver diseases, decompensated; Ulcerative colitis; Crohn’s disease; dietary iron deficiency; other infectious diseases; other neglected tropical diseases; other endocrine, nutrition, blood, and immune disorders; and other hemoglobinopathies and hemolytic anemias.
+
+The following table displays the cause- and sex-specific hemoglobin shifts used for the anemia causal attribution process in GBD 2019.
```

Comment:
Let's put a reference here, to link directly to the source. Are these numbers actually presented in published materials?

Reply:
These numbers are not referenced in the GBD methods appendices and are also not sourced on the anemia team stash code, but rather were sent to me directly from Jordan Bailey, the iron deficiency modeler.

Reply:
oh, I was wrong! They are hosted here: https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/priors/hb_shifts.csv

I will link this page